### PR TITLE
Fixes Readme and changes for netlify deployment

### PR DESCRIPTION
**Pull request by jeromesimeon**
- Updates to the READMEs
- Fixes root package.json for netlify deployment

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "react-dom": "^16.11.0"
   },
   "scripts": {
-    "install": "npm run bootstrap",
+    "install": "npm install lerna && npm run bootstrap",
```
Comment (mttrbrts) [inline package.json]: Instead I would add `lerna` as a dependency, keep this script the same

**Review by mttrbrts — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (jeromesimeon) [reply package.json]: Another approach might be to remove `install` altogether. This trips netlify. Do we need it for something?

Comment (mttrbrts) [reply package.json]: Yes, I think that we still need to cause `install` to actually trigger `lerna bootstrap`

Comment (jeromesimeon) [reply package.json]: > Yes, I think that we still need to cause `install` to actually trigger `lerna bootstrap`

I fixed the netlify deploy to do `npm install -g lerna && lerna bootstrap && npm run build` and it seems to work!

Comment (jeromesimeon) [reply package.json]: by default netlify calls `npm install` if present.

Comment (mttrbrts) [reply package.json]: But without an `install` script, netlify won't know to call `lerna bootstrap`, right?

Comment (jeromesimeon) [reply package.json]: This seems to work:

Screenshot 2020-02-24 at 9 46 12 AM

It was already there so I assumed it was your idea in the first place? It was just not working with `install` which is called before the build command.

Comment (mttrbrts) [reply package.json]: oh, sorry
! It's all good then!

Comment (jeromesimeon) [reply package.json]: Proof is in the pudding: netlify link is below in this PR.

**Review by mttrbrts — APPROVED** [review]

**Pull request merged**